scripts/performance/common.py

```diff
@@ -87,6 +87,28 @@ def get_artifacts_directory() -> str:
     '''
     return os.path.join(get_repo_root_path(), 'artifacts')
 
+def rename_upload_files(files) -> None:
```

Comment:
Is this function going to be used outside `upload.py`?

Reply:
Yes. It is used in azcopy.py.